docs/DETAILED_INSTALL.md

```diff
@@ -1,8 +1,38 @@
 # Detailed installation guide
 
+### Conda virtual environment
+We recommend using `conda` for package management. These instructions should allow you to install and run mava.
+
+1. Create and activate a virtual environment
+```bash
+conda create -n mava python=3.12
+conda activate mava
+```
+
+2. Clone mava
+```bash
+git clone https://github.com/instadeepai/Mava.git
+cd mava
+```
+
+3. Install the dependencies
+```bash
+pip install -e .
+```
+
+4. Install jax on your accelerator. The example below is for an NVIDIA GPU, please the [official install guide](https://github.com/google/jax#installation) for other accelerators
+```bash
+pip install -U "jax[cuda12]==0.4.25"
```

Comment:
I think for 0.4.25 we should rather say this:

```suggestion
pip install "jax[cuda12_pip]==0.4.25" -f https://storage.googleapis.com/jax-releases/jax_cuda_releases.html"
```

We can also take away the upgrade flag. Since `JAX` doesn't pin the `numpy` version we will accidentally upgrade `numpy` here.